package.json

```diff
@@ -34,7 +34,7 @@
     "@data-driven-forms/pf3-component-mapper": "~1.30.0",
     "@data-driven-forms/react-form-renderer": "~1.30.0",
     "@manageiq/react-ui-components": "~0.11.68",
-    "@manageiq/ui-components": "~1.4.0",
+    "@manageiq/ui-components": "1.4.1",
```

Comment:
missing `~`, master doesn't yarn.lock so with the tilde it's possible to update

Reply:
Fixed.